src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background: url(./images/star.svg) no-repeat center;
+  margin-right: 4px;
+}
+
+.stars_star:last-child {
+  margin-right: 0;
+}
```

It's better to use like this) 
```suggestion
}

.stars_star:not(:last-child) {
    margin-right: 4px;
}
```

---

I have just corrected